### PR TITLE
Dont require frame formulation before EU exit

### PR DESCRIPTION
**Pull request by slawosz**
This also fixes `was_notified_before_eu_exit` for notification upload.

### CODE REVIEW

**Comment by nasirkhanbeis** [comment]
Just a minor issue, otherwise all good.
when formulation document is uploaded although it is not mandatory, we need to show in summary page when notified.

**Review by nasirkhanbeis — APPROVED** [review]
looks good as expected.

File: cosmetics-web/app/models/notification.rb [hunk 1]
```diff
@@ -147,7 +147,7 @@ def notified_post_eu_exit?
   alias_method :requires_images?, :notified_post_eu_exit?
 
   def notified_pre_eu_exit?
-    was_notified_before_eu_exit? || (cpnp_notification_date.present? && (cpnp_notification_date < EU_EXIT_DATE))
+    was_notified_before_eu_exit?
   end
```
Comment (scruti) [inline cosmetics-web/app/models/notification.rb:151]: It does not seem that we need to have `notified_pre_eu_exit?` anymore now that we can replace the method with `was_notified_before_eu_exit?` calls.

File: cosmetics-web/app/services/cpnp_notification_importer.rb [hunk 2]
```diff
@@ -23,7 +23,8 @@ def create!
                                         still_on_the_market: @cpnp_parser.still_on_the_market,
                                         components_are_mixed: @cpnp_parser.components_are_mixed,
                                         ph_min_value: @cpnp_parser.ph_min_value,
-                                        ph_max_value: @cpnp_parser.ph_max_value)
+                                        ph_max_value: @cpnp_parser.ph_max_value,
+                                        was_notified_before_eu_exit: (@cpnp_parser.cpnp_notification_date < EU_EXIT_DATE))
```
Comment (scruti) [inline cosmetics-web/app/services/cpnp_notification_importer.rb:27]: Neat :1st_place_medal:

**Review by scruti — APPROVED** [review]
LGTM. The only small issue is that one method (`Notification#notified_pre_eu_exit`) seems to have become redundant.

**Review by scruti — APPROVED** [review]
Re-approved

**Comment by nasirkhanbeis** [comment]
re-tested code changes, looks good.

**Pull request merged**